src/Imaging/GlideManager.php

```diff
@@ -141,8 +141,12 @@ public function clearAsset($asset)
         $pathPrefix = ImageGenerator::assetCachePathPrefix($asset);
         $manifestKey = ImageGenerator::assetCacheManifestKey($asset);
 
-        // Delete generated glide cache for asset.
-        $this->server()->deleteCache($pathPrefix.'/'.$asset->path());
+        // Delete generated glide cache for asset, making sure to use the default cache path callable.
```

Comment:
Seems like tests are failing as that method isnt in the earlier versions of glide that the test suite is using. Maybe something like:

 ```
$server = $this->server();

 if (method_exists($server, 'getCachePathCallable)) {
        $customCallable = $server->getCachePathCallable();
        $server->setCachePathCallable(null);
        $server->deleteCache($pathPrefix.'/'.$asset->path());
        $server->setCachePathCallable($customCallable);
 } else {
         $server->deleteCache($pathPrefix.'/'.$asset->path());
}
```

Reply:
@ryanmitchell That's a good point. I've now wrapped that part in a conditional, same as the part above.